src/libraries/System.Data.OleDb/src/OleDbConnection.COMWrappers.cs

```diff
@@ -0,0 +1,99 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.Data.Common;
+using System.Data.ProviderBase;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Data.OleDb
+{
+    using SysTx = Transactions;
+
+    public sealed partial class OleDbConnection
+    {
+        internal static Exception? ProcessResults(OleDbHResult hresult, OleDbConnection? connection, object? src)
```

Comment:
Can we share the bulk of this code, and only pull out the differences into `COMWrappers` vs. `NoCOMWrappers` files? i.e. create new private methods on the partial classes.